kube_deploy/Dev/deploy.yml

```diff
@@ -79,6 +79,11 @@ spec:
                 secretKeyRef:
                   name: cica-web-secrets
                   key: cw_os_places_api_key
+            - name: CW_SESSION_DURATION
```

Comment:
Add this is a separate commit to make it clear the intent behind this change

Reply:
this was a goof